bedrock/firefox/firefox_details.py

```diff
@@ -12,6 +13,11 @@
 class _ProductDetails(ProductDetails):
     bouncer_url = 'https://download.mozilla.org/'
 
+    # Note allizom.org is the production endpoint for SHA-1.
+    # It uses this because that's the only SHA-1 certificate
+    # we have that's usable. (SHA-1 certs can no longer be issued).
+    sha1_bouncer_url = 'https://download.mozilla.org/' if switch('disable-sha1-downloads') else 'https://download-sha1.allizom.org/'
```

Comment:
AH! DEV! Of course you're right. I'd forgotten. Please ignore me here and do the `get_bouncer_url()` thing or the like.